Classes/RoutingComponent.php

```diff
@@ -34,6 +38,18 @@ class RoutingComponent extends \Neos\Flow\Mvc\Routing\RoutingComponent
      */
     protected $router;
 
+    /**
+     * @var UriFactoryInterface
+     * @Flow\Inject
+     */
+    protected $uriFactory;
+
+    /**
+     * @var ResponseFactoryInterface
+     * @Flow\Inject
+     */
+    protected $responseFactory;
```

Comment:
Do we actually need this?

Reply:
well... I thought I'll use the factory. But it would be possible without as well. May I ask what your concern his?

Reply:
You don't use it in the code at all, right? So it should be "useless" as it is. I'm cool with using it, but right now there is a `new Response()`

Reply:
Oh.. I don't? Checking...